tests/testthat/test-Design-methods.R

```diff
@@ -92,3 +92,99 @@ test_that("NextBestInfTheory produces consistent results with a dataset", {
   expect_equal(result@toxAtDosesSelected, rep(1L, 5))
   expect_snapshot(result@meanFit)
 })
+
+test_that("reporting labels and logicals are correctly returned using the recursive function in simulate", {
+  my_data <- h_get_data(placebo = FALSE)
```

Comment:
```suggestion
  data <- h_get_data(placebo = FALSE)
```
In unit tests we do not use `my_` prefix.